src/tools/ScenarioMeasurement/Startup/ParserUtility.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Microsoft.Diagnostics.Tracing;
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Data;
+using System.Data.Common;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+
+namespace ScenarioMeasurement
+{
+    class ParserUtility
+    {
+
+        public static bool MatchProcessStart(TraceEvent evt, TraceSourceManager source, string processName, IList<int> pids, string commandLine)
+        {
+            return MatchCommandLine(evt, source, commandLine) &&
+                   MatchProcessName(evt, source, processName) &&
+                   MatchProcessID(evt, source, pids);
+        }
+
+        public static bool MatchCommandLine(TraceEvent evt, TraceSourceManager source, string commandLine)
+        {
+            if (source.IsWindows)
+            {
+                if (((string)GetPayloadValue(evt, "CommandLine")).Trim() != commandLine)
```

Comment:
We should make this compare look like all the other ones that we do in this file.

Reply:
Oh do you mean using "Equals"?

Reply:
Yeah, seems like it would be nice to do the compares similarly